src/libcore/slice/sort.rs

```diff
@@ -6,12 +6,12 @@
 //! Unstable sorting is compatible with libcore because it doesn't allocate memory, unlike our
 //! stable sorting implementation.
 
-// ignore-tidy-undocumented-unsafe
-
 use crate::cmp;
 use crate::mem::{self, MaybeUninit};
 use crate::ptr;
 
+// ignore-tidy-undocumented-unsafe
```

Comment:
Please don't move this comment below the `use` lines like this. See the overall diff; these two lines shouldn't change at all.